### PR TITLE
Url branding for organization authenticator

### PR DESCRIPTION
**Pull request by sadilchamishka**
## Purpose
$subject

## Related Issues
- https://github.com/wso2/product-is/issues/14099

### CODE REVIEW

**Review by madurangasiriwardena — APPROVED** [review]

File: components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java [hunk 1]
```diff
@@ -341,6 +350,18 @@ public List<String> getAncestorOrganizationIds(String organizationId) throws Org
...
+    @Override
+    public List<Organization> getOrganizationsByName(String organizationName)
+            throws OrganizationManagementException {
+
+        List<Organization> organizations = organizationManagementDAO.getOrganizationsByName(organizationName);
+        if (CollectionUtils.isNotEmpty(organizations)) {
+            return organizations;
+        } else {
+            throw handleClientException(ERROR_CODE_RETRIEVING_ORGANIZATIONS_BY_NAME, organizationName);
+        }
```
Comment (chamathns) [inline components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java:362]: else is redundant

Comment (sadilchamishka) [reply components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java:362]: The codebase is moved to a new repository, hence the improvement is send as PR for the particular repository.            
- https://github.com/wso2-extensions/identity-organization-management-core/pull/30

File: components/org.wso2.carbon.identity.organization.management.authn/src/main/java/org/wso2/carbon/identity/organization/management/application/authn/EnterpriseIDPAuthenticator.java [hunk 2]
```diff
@@ -288,6 +310,32 @@ private void redirectToOrgNameCapture(HttpServletResponse response, Authenticati
         }
     }
 
+    @SuppressFBWarnings(value = "UNVALIDATED_REDIRECT", justification = "Redirect params are not based on user inputs.")
+    private void redirectToSelectOrganization(HttpServletResponse response, AuthenticationContext context,
```
Comment (ImalshaG) [inline components/org.wso2.carbon.identity.organization.management.authn/src/main/java/org/wso2/carbon/identity/organization/management/application/authn/EnterpriseIDPAuthenticator.java:314]: Shall we add method descriptions?

Comment (sadilchamishka) [reply components/org.wso2.carbon.identity.organization.management.authn/src/main/java/org/wso2/carbon/identity/organization/management/application/authn/EnterpriseIDPAuthenticator.java:314]: The codebase is moved to a new repository, hence the improvement is send as a PR for the particular repository.            - https://github.com/wso2-extensions/identity-auth-organization-login/pull/16

**Pull request merged**